pkg/printers/internalversion/describe.go

```diff
@@ -893,6 +893,15 @@ func printGlusterfsVolumeSource(glusterfs *api.GlusterfsVolumeSource, w PrefixWr
 		glusterfs.EndpointsName, glusterfs.Path, glusterfs.ReadOnly)
 }
 
+func printGlusterfsPersistentVolumeSource(glusterfs *api.GlusterfsPersistentVolumeSource, w PrefixWriter) {
+	w.Write(LEVEL_2, "Type:\tGlusterfs (a Glusterfs mount on the host that shares a pod's lifetime)\n"+
+		"    EndpointsName:\t%v\n"+
+		"    EndpointsNamespace:\t%v\n"+
+		"    Path:\t%v\n"+
+		"    ReadOnly:\t%v\n",
+		glusterfs.EndpointsName, glusterfs.EndpointsNamespace, glusterfs.Path, glusterfs.ReadOnly)
```

Comment:
do you want to  print `nil` for EndpointsNamespace or show the PVC namespace?

Reply:
I was thinking  that, it can be nil or a namespace which is in the spec. Are you saying its going to be `nil` always ?

Reply:
it's nil by default... I guess we don't have the namespace of the pvc here to display in that case